### ISSUE TITLE
Bootstrap4: When using category tabs, content of first tab is empty

### ISSUE THREAD

**Issue by JElchison** [json-editor/json-editor#681]
_Thanks to the many who have made this awesome project happen!_

### General information

- json-editor version:  2.1.0

### Expected behavior
When using category tabs, content of first tab is displayed upon page load

### Actual behavior
When using category tabs, content of first tab is empty upon page load:

![image](https://user-images.githubusercontent.com/1757771/76170182-e3c9f780-6155-11ea-8d43-bf505dfbf6e2.png)

Content appears all at once when clicking on the tab name:

![image](https://user-images.githubusercontent.com/1757771/76170191-efb5b980-6155-11ea-81e2-e501adabc48f.png)

This issue occurs regardless of the number of category tabs.

### Steps to reproduce the behavior
Direct link to example: https://json-editor.github.io/json-editor/?data=N4Ig9gDgLglmB2BnEAuUMDGCA2MBGqIAZglAIYDuApomALZUCsIANOHgFZUZQD62ZAJ5gArlELwwAJzplsrEIgwALKrNShYUbFUIBhMoirYwZACYKogiLpTsuPBSRllxdjK6oBzaTBoKIKUgqKVh/NBAzGEQoKUw3TRhtWxAAEWjY+MtrFLBObnE2QODQv2QImLJQ3jNPDRAtHUIAMTAwbJtCGLj4LwU6GHgAGSpeqGVCAEYQAF85ubZEZTAKXhCgqXKQQagQsh44eEtVBkI8NqhusggAFlmZoA=

Alternatively, visit https://json-editor.github.io/json-editor/, and use the below schema:

```javascript
{
  "title": "Caseload",
  "type": "object",
  "format": "categories",
  "properties": {
    "district": {
      "title": "District",
      "type": "object",
      "properties": {
        "start_date": {
          "title": "Foo",
          "type": "string",
          "minLength": "1"
        }
      }
    }
  }
}
```

Default options are sufficient.  My application is using bootstrap4 and fontawesome5.

Click "Update Schema" button.

Tested in multiple browsers:
* Chrome Version 80.0.3987.122 (Official Build) (64-bit)
* Firefox 73.0.1 (64-bit)

All tests conducted on:
```
$ uname -a 
Linux xxxxxxxx 4.15.0-88-generic #88-Ubuntu SMP Tue Feb 11 20:11:34 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.4 LTS
Release:	18.04
Codename:	bionic
```

_Thanks in advance for any insight on what I might be doing wrong._

**Comment by VadimTikhonyuk** [comment]
I have same issue, have to click on it and then it become visible

**Comment by JElchison** [comment]
More info:  I was able to ascertain that this regression was introduced between tags [2.0.0-alpha.0](https://github.com/json-editor/json-editor/releases/tag/2.0.0-alpha.0) and [2.0.0-alpha.1](https://github.com/json-editor/json-editor/releases/tag/2.0.0-alpha.1).

For convenience, here is a [diff between the tags](https://github.com/json-editor/json-editor/compare/2.0.0-alpha.0...2.0.0-alpha.1).

**Comment by JElchison** [comment]
To narrow the search further from a black-box perspective, I ran `git bisect` against the tree between the two alpha versions listed above.  Result:
```
f5351ca (bad)  bootstrap styled tooltips  Jonathan Ströbele, 5 months ago
989fb79 (skip) tabs correctly nested  Jonathan Ströbele, 5 months ago
65d52b5 (skip) table styles + tweaking for form-group in cells  Jonathan Ströbele, 5 months ago
bcf94d5 (skip) update to latest form feedback classes  Jonathan Ströbele, 5 months ago
745675e (skip) show top tab as card with nav  Jonathan Ströbele, 5 months ago
17e054b (skip) panel spacing below  Jonathan Ströbele, 5 months ago
b2b588d (skip) button group spacing  Jonathan Ströbele, 5 months ago
2677892 (good) twbs4 radios stacked / inline  Jonathan Ströbele, 5 months ago
```

It appears that one of these commits by @stroebjo is likely at fault.

Here is a diff that most likely contains the cause of this regression:  https://github.com/json-editor/json-editor/compare/2677892...f5351ca

It appears that this regression is specific to Bootstrap4.

@stroebjo Are you able to lend a hand debugging this issue?  Thanks in advance!

**Comment by stroebjo** [comment]
Ah. We have discusses this issue a few month ago, but didn't produce a fix then https://github.com/json-editor/json-editor/pull/588#issuecomment-547090854 and https://github.com/json-editor/json-editor/issues/594.

It happens for the category type, but not the "tab-top" one.

I had a look at it again, and the commit that introduced the bug is https://github.com/json-editor/json-editor/commit/745675ed197c480a96dcdfab6a1b791ed6b5aa32.

To get the TWBS "card" style I added new `div.card-header` and `div.card-body` wrappers in the `getTopTabHolder()` function and accordingly changed the returned node in `addTopTab()` and `getTopTabContentHolder()` so the core can append the tab link and body. 

If you revert this change / remove the additional layers it works again - but you lose the TWBS card style. 

I'm not really sure why this happens, maybe one of the core maintainers can have a look?

**Comment by JElchison** [comment]
Thanks for the speedy response, @stroebjo!  Also, thanks for your contributions to this great project.

I have verified that reverting 745675e indeed works around this issue.

I have tried to read up on the histories for #588 and #594, and done some debugging of my own.  I have found that, for some reason, in Bootstrap4's `markTabActive()`, `row.rowPane` is undefined, and thus never set to `active`.  Upon initial page load, no pane is set with class `active` (which is commensurate with the [SO post referenced](https://stackoverflow.com/questions/52582990/bootstrap-4-navigation-tabs-content-not-showing-content-on-initial-page-load)).

https://github.com/json-editor/json-editor/blob/a7f1b9b57e257ad32446492c99e63bd426d61686/src/themes/bootstrap4.js#L489

I have stepped through page initialization, and I see `rowPane` being set to a valid value.  Does anyone have any ideas for why the value may be squashed?
